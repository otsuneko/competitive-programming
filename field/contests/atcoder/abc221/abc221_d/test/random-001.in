5
2 2
2 6
9 10
5 8
9 7
