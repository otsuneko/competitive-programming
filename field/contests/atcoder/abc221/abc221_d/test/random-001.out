5 8 4 0 0
